openapi2kong/openapi2kong.go

```diff
@@ -377,7 +377,8 @@ func getPluginsList(
 	}
 
 	if extensions == nil && componentExtensions == nil {
-		return nil, nil
+		emptyList := make([]*map[string]interface{}, 0)
```

Comment:
We are choosing to avoid returning null when there is no error - so that all consumers don't have to do null check.

Reply:
Can we add this as a comment above the same line?